clients/python/test/lib.py

```diff
@@ -15,7 +15,7 @@ def lib_test_available_grpc(model_name, server_grpc):
     assert client._is_model_ready() is None
 
 
-def lib_test_inference(model_name, server_grpc, atol=1e-6):
+def lib_test_inference(model_name, server_grpc, atol=1e-4):
```

Comment:
Please revert this and change the atol in the function call of the models where you need it.

Reply:
Changed default value back to `atol=1e-6` and only overwriting it to `atol=1e-4` for my own function call